tests/docker_scripts/fasttest_runner.sh

```diff
@@ -233,10 +233,8 @@ function build
         ) | ts '%Y-%m-%d %H:%M:%S' | tee -a "$FASTTEST_OUTPUT/test_result.txt"
 
         if [ "$COPY_CLICKHOUSE_BINARY_TO_OUTPUT" -eq "1" ]; then
-            mkdir -p "$FASTTEST_OUTPUT/binaries/"
-            cp programs/clickhouse "$FASTTEST_OUTPUT/binaries/clickhouse"
-
-            zstd --threads=0 programs/clickhouse-stripped -o "$FASTTEST_OUTPUT/binaries/clickhouse-stripped.zst"
+            cp programs/clickhouse "$FASTTEST_OUTPUT/clickhouse"
```

Comment:
no no no no no, this would put it to the reports bucket. We don't  want it, I believe

I don't know how to add artifact to the new CI, but it should be somewhere in `ci/workflows/defs.py`

Reply:
How it is different? I think previously it was also in the reports bucket

Reply:
UPD: I've just looked at bash_history, and no, it was in builds - `wget https://s3.amazonaws.com/clickhouse-builds/PRs/62123/b742df7fd712066141f2c119c964bf6c5a14e5c2/fast_test/clickhouse-stripped.zst`

Reply:
@Felixoid can you help with this?